src/app/identity.service.ts

```diff
@@ -0,0 +1,62 @@
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Inject, Injectable, InjectionToken } from '@angular/core';
+import { TransferState } from '@angular/platform-browser';
+import { StorageService } from 'ngx-webstorage-service';
+import { Observable } from 'rxjs';
+
+import { AnalyticsService } from './analytics.service';
+import { environment } from '../environments/environment';
+import { Person } from './person.model';
+
+export const TBG_DONATE_ID_STORAGE = new InjectionToken<StorageService>('TBG_DONATE_ID_STORAGE');
+
+@Injectable({
+  providedIn: 'root',
+})
+export class IdentityService {
+  private readonly apiPath = '/people';
+  // Key is per-domain/env. For now we simply store a single JWT (or none).
+  private readonly storageKey = `${environment.identityApiPrefix}/v1/jwt`;
```

Comment:
@NoelLH The `/v1` here is extra, I think. You've already included `v1` as part of the environment variable definition. See the environment files.

Reply:
@alihejazi This key is not a URL, but an identifier so we can tell with no risk of clashes which local data works with which version of the frontend. It's desirable to have "v1" to designate that this is the first way of using local storage because we should be able to change the client implementation without publishing a new API: the two are independent.

I think this syntax is how Donate already works which the other storage example, so I think that making a different structure (e.g. dot separators) at this point is probably more confusing than just sticking with this.